### PR TITLE
DOC: fixes #916

### LINKED ISSUE #916: Installation ubuntu 14.04
I had some issues tryint to start up qiita 
```
Traceback (most recent call last):
  File "/home/jorge/anaconda/envs/qiita/bin/qiita_env", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/home/jorge/code/qiita/scripts/qiita_env", line 154, in <module>
    env()
  File "/home/jorge/anaconda/envs/qiita/lib/python2.7/site-packages/click/core.py", line 384, in __call__
    return self.main(*args, **kwargs)
  File "/home/jorge/anaconda/envs/qiita/lib/python2.7/site-packages/click/core.py", line 370, in main
    self.invoke(ctx)
  File "/home/jorge/anaconda/envs/qiita/lib/python2.7/site-packages/click/core.py", line 614, in invoke
    return self.invoke_subcommand(ctx, cmd, cmd_name, ctx.args[1:])
  File "/home/jorge/anaconda/envs/qiita/lib/python2.7/site-packages/click/core.py", line 623, in invoke_subcommand
    return cmd.invoke(cmd_ctx)
  File "/home/jorge/anaconda/envs/qiita/lib/python2.7/site-packages/click/core.py", line 524, in invoke
    ctx.invoke(self.callback, **ctx.params)
  File "/home/jorge/anaconda/envs/qiita/lib/python2.7/site-packages/click/core.py", line 236, in invoke
    return callback(*args, **kwargs)
  File "/home/jorge/code/qiita/scripts/qiita_env", line 71, in make
    make_environment(load_ontologies, download_reference, add_demo_user)
  File "/home/jorge/code/qiita/qiita_db/environment_manager.py", line 180, in make_environment
    admin_conn = SQLConnectionHandler(admin='admin_without_database')
  File "/home/jorge/code/qiita/qiita_db/sql_connection.py", line 120, in __init__
    self._open_connection()
  File "/home/jorge/code/qiita/qiita_db/sql_connection.py", line 155, in _open_connection
    raise RuntimeError("Cannot connect to database: %s" % str(e))
RuntimeError: Cannot connect to database: fe_sendauth: no password supplied
```
and it was solved by setting a password:
```
$ psql postgres
ALTER USER postgres PASSWORD 'supersecurepassword';
\q
```
and updating the qiita_config.txt file accordingly.

### PR DESCRIPTION
**Pull request by Jorge-C**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -55,6 +55,32 @@ After these commands are executed, you will need to:
 
 If all the above commands executed correctly, you should be able to go to http://localhost:21174 in your browser, to login use `demo@microbio.me` and `password` as the credentials. (In the future, we will have a *single user mode* that will allow you to use a local Qiita server without logging in. You can track progress on this on issue [#920](https://github.com/biocore/qiita/issues/920).)
 
+## Installation issues on Ubuntu 14.04
+
+### `fe_sendauth: no password supplied`
+
+If you get a traceback similar to this one when starting up qiita
```
Comment (ElDeveloper) [inline INSTALL.md]: qiita -> Qiita

File: INSTALL.md [hunk 2]
```diff
@@ -55,6 +55,32 @@ After these commands are executed, you will need to:
...
+```
```
Comment (ElDeveloper) [inline INSTALL.md]: Add `python` at the end of the line.

**Comment by ElDeveloper** [comment]
Looks good, very minor problems. Can you also add `[skip ci]` to your next commit message? That way Travis will not execute the test suite as this is purely a documentation change.

**Comment by Jorge-C** [comment]
Thanks! I fixed your comments. Good suggestion about skipping CI.

2015-03-12 10:44 GMT-06:00 Yoshiki Vázquez Baeza <notifications@github.com>:

**Comment by ElDeveloper** [comment]
:+1:

**Pull request merged**